### DISCUSSION TITLE
Django_tenants_q2

### DISCUSSION THREAD

**Discussion by mast3rbow** [django-q2/django-q2#117] [Show and tell]
Hi everyone,

I've re-written the old django_tenants_q which has been stale to now leverage the Django Q2 improvements. 

https://github.com/mast3rbow/django_tenants_q

If you find improvements, please don't hesitate to make PR improvements!

**Comment by mast3rbow** [comment]
@GDay would it be possible to connect sometime, would be keen to ensure I have updated this package correctly.

**Reply by GDay** [reply]
Not sure what you need from me exactly, but you are free to send me an email: hello@chiefonboarding.com
